manifests/params.pp

```diff
@@ -189,7 +189,7 @@
   # Application logging
   $logging_level = 'info'
   $logging_type = 'file'
-  $logging_layout = 'pattern'
+  $logging_layout = 'multiline_request_pattern'
```

Comment:
Thinking more about it, it wouldn't be backwards incompatible if we kept this the same. There's a migration in the installer anyway so it isn't needed there.

Reply:
makes sense

Reply:
done with https://github.com/theforeman/puppet-foreman/pull/904